test/jest-api.setup.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
 
 const isoDateRegex = /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z/;
-
-jest.setTimeout(60000);
+process.env.DATABASE_NAME = `strapi_${process.env.JEST_WORKER_ID}`;
```

Comment:
Can't we use different schemas instead of using multiple DBs? 🤔 What is the benefit of having multiple DBs?

Reply:
Did not think about using schemas this could maybe be the breakthrough to be able to remove the root complexity.

Reason I did different dbs is if you do it all on the same db whiteout different schemas the different workers will start fighting over what tables will need to be in the db and tests start randomly failing

I will take a look if I can use schemas  instead of different dbs